### DISCUSSION TITLE
See all possible achievements.

### DISCUSSION THREAD

**Discussion by Exortions** [community/community#18544] [Other Feature Feedback, Questions, & Ideas]
First, I enjoy the new achievements feature on GitHub, and I think it was a great addition.

However, (And I've noticed this in video games too), not being able to view the *possible* achievements is a big downside. Many people will want to get as many achievements as possible, and adding a feature to view all achievements would be highly beneficial.

I also fully understand that you may not want to implement this feature and that you want users to find achievements on their own.

**Comment by lgrachov** [comment]
Hmmm, as I known. In Minecraft you can view badges

**Reply by happydev000** [reply]
What's that? Could you share the link?

**Reply by PythonCoderAS** [reply]
I don't think there are badges in normal Minecraft. Perhaps you mean Achievements (Bedrock) or Advancements (Java)?

**Comment by number0x0001** [comment]
There's an unofficial one here: https://github.com/Schweinepriester/github-profile-achievements.

Github used to have an official one at https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile, but it's not updated.
